dali/ft/filecopy.cpp

```diff
@@ -3026,6 +3026,7 @@ bool FileSprayer::isSameSizeHeaderFooter()
 
 void FileSprayer::updateTargetProperties()
 {
+    LOG(MCdebugInfo, job, "FileSprayer::updateTargetProperties() started");
```

Comment:
left in accidentally  I assume?

Reply:
No, left it intentionally to measure if any further things happen.

Reply:
ok, is once per spray?

Reply:
Yes, that is the last step to copy/update/create metainfo.